stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
```

Comment:
Maybe. I am not sure whether we even want to have the helper functions, or just call `responseObserver.onError(Status.UNIMPLEMENTED.asException())` and return a NOOP StreamObserver directly from the generated method body.